assets/src/models/notificationData.ts

```diff
@@ -0,0 +1,27 @@
+import { Notification, NotificationReason, RunId } from "../realtime.d"
+import { RouteId, TripId } from "../schedule.d"
+import { dateFromEpochSeconds } from "../util/dateTime"
+
+export interface NotificationData {
+  created_at: number
+  reason: NotificationReason
+  route_ids: RouteId[]
+  run_ids: RunId[]
+  trip_ids: TripId[]
+}
+
+export const notificationsFromData = (
+  notificationsData: NotificationData[]
+): Notification[] => notificationsData.map(notificationFromData)
+
+const notificationFromData = (
+  notificationData: NotificationData,
+  index: number
+): Notification => ({
+  id: Date.now() + index,
```

Comment:
@skyqrose I've thought this through and it's a safe way to generate these indices, but I did have to think it through, so let's maybe document the reasoning: that busloc pushes new block waivers to Skate only every few seconds, meaning that a collision of IDs here could only happen if thousands of block waivers are generated per second for multiple seconds running.